src/main/java/com/google/sps/servlets/MentorSignupServlet.java

```diff
@@ -71,7 +106,7 @@ private void addMentorExperience(String[] experienceTags) {
 
       try {
         // Establish connection to MySQL database.
-        Connection connection = DriverManager.getConnection(url, user, password);
+        Connection connection = DriverManager.getConnection(Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
```

Comment:
Good call with updating these to use the consts in the Utility class! But also make sure you clean up the unused local variables above :) (lines 98-101)

Reply:
Oh, right! Thanks.